formfyxer/lit_explorer.py

```diff
@@ -1270,16 +1305,16 @@ def weight(stats, metric):
         weight = metric.get("weight") or 1
         val = 0
         if "clip" in metric:
-            val = min(max(stats[name], metric["clip"][0]), metric["clip"][1])
-        elif isinstance(stats[name], bool):
-            val = 1 if stats[name] else 0
+            val = min(max(stats.get(name,0), metric["clip"][0]), metric["clip"][1])
+        elif isinstance(stats.get(name), bool):
+            val = 1 if stats.get(name) else 0
```

Comment:
Can we take these changes back out? My idea was that I'd rather have the app crash if we are using the wrong name or spelling of a metric name, and going through parse_form once with one form is enough to catch these errors. It's much easier to see the crash and fix it than to realize we've been misspelling things for a month and some values have been silently turning into 0s.

Reply:
My thought was I didn't want to break old links to cached stats

Reply:
That's fair, but we should just recompute cached stats in that case.